### ISSUE TITLE
[FEATURE] Add more Social Links

### ISSUE THREAD

**Issue by nathgoutam93** [nathgoutam93/linkgen-react#3]
### Description

Right now only a few Social Links are supported for links as icons . Add more Popular social platform .

The links should be added to Social Icons card , Page and Static Page component .

**Comment by sauradip007** [comment]
Hi , I would like to work on this issue as I've just started learning react and components. My understanding of the problem is there are more social media links and icons to be added here:

![image](https://user-images.githubusercontent.com/71842251/155695393-8c1ad683-0c8e-469e-bbce-7ca7cf7b195c.png)

The same can be implemented by working the code as provided in the SocialIconCard.js, page.js and staticPage.js files  for more such social media platforms say for example dev.to . Please guide me for the same

**Comment by nathgoutam93** [comment]
> Hi , I would like to work on this issue as I've just started learning react and components. My understanding of the problem is there are more social media links and icons to be added here:
> 
> ![image](https://user-images.githubusercontent.com/71842251/155695393-8c1ad683-0c8e-469e-bbce-7ca7cf7b195c.png)
> 
> The same can be implemented by working the code as provided in the SocialIconCard.js, page.js and staticPage.js files for more such social media platforms say for example dev.to . Please guide me for the same

Yes , the code structure is already there to follow if you would like to work on this . 

Go for it . @sauradip007 if you need any help feel free to ask

**Comment by nathgoutam93** [comment]
@sauradip007 thanks for your contribution 🙌

**Comment by nathgoutam93** [comment]
Pull requests are welcome if anyone would like to include other Socials as well

**Comment by sauradip007** [comment]
np , also I'd like to apologise for not framing the PR template properly and my double branch commit 😛 . I'd like to contribute more if you could be more specific about what kind of platforms you'd want as links like if its just limited to social platforms only.

**Comment by nathgoutam93** [comment]
> np , also I'd like to apologise for not framing the PR template properly and my double branch commit 😛 . I'd like to contribute more if you could be more specific about what kind of platforms you'd want as links like if its just limited to social platforms only.

That's why I'm keeping it open cause I can't tell you specifically the name of the platforms from top of my head 😅 I would say you can look at other link in Bio' services what socials they are offering

**Comment by vish-han** [comment]
@nathgoutam93  can I work on this issue, I'm currently learning react and it will help my skills also.

**Comment by nathgoutam93** [comment]
> @nathgoutam93 can I work on this issue, I'm currently learning react and it will help my skills also.

sure go for it, but please mention the socials you are going to add before adding, as most of the socials are already added I cannot come up with more suggestions to give.

**Comment by vish-han** [comment]
I was thinking of adding a icon which redirect to the personal portfolio of the user if they have any....

what your view on this??

**Comment by nathgoutam93** [comment]
> I was thinking of adding a icon which redirect to the personal portfolio of the user if they have any....
> 
> what your view on this??

Portfolio is not what I'd like to have as an icon as it has no specific icon, I'd like to have email, YouTube, TikTok, SnapChat etc

as most will like to add Portfolio link with  a description, So I think they can use standard link for that.

**Comment by nathgoutam93** [comment]
you can add the socials I mentioned though , I'd happy to assign you if you are willing to add those @vish-han

**Comment by vish-han** [comment]
Yeah I'm willing to work on youtube icon first😁

**Comment by nathgoutam93** [comment]
sure go for it

**Comment by senali-d** [comment]
@nathgoutam93 can I work on this issue.

**Comment by nathgoutam93** [comment]
> @nathgoutam93 can I work on this issue.

Sure, go for it

**Comment by senali-d** [comment]
@nathgoutam93 Suggestion for social link - hashnode.

**Comment by nathgoutam93** [comment]
> @nathgoutam93 Suggestion for social link - hashnode.

Hashnode is already added.
